_posts/fr/newsletters/2023-09-27-newsletter.md

```diff
@@ -0,0 +1,211 @@
+  de spécifier une clé publique et le nonce en deux parties [MuSig2][topic musig]. {% assign timestamp="50:18" %}
+
+- [LDK #2547][] met à jour le code de recherche de chemin probabiliste en supposant qu'il est plus probable que les canaux distants
+  aient la plupart de leur liquidité poussée d'un côté du canal. Par exemple, dans un canal distant de 1,00 BTC entre Alice et Bob,
+  l'état le moins probable du canal est que Alice et Bob aient chacun 0,50 BTC. Il est plus probable que l'un d'entre eux ait 0,90
+  BTC---et encore plus probable que l'un d'entre eux ait 0,99 BTC. {% assign timestamp="51:10" %}
+
+- [LDK #2534][] ajoute la méthode `ChannelManager::send_preflight_probes` pour sonder les chemins de paiement avant d'essayer d'envoyer
+  un paiement. Une sonde est générée par un expéditeur comme un paiement LN régulier, mais la valeur de son préimage [HTLC][topic htlc]
+  est définie sur une valeur inutilisable (par exemple, une valeur connue uniquement de l'expéditeur) ; lorsqu'elle atteint sa
+  destination, le destinataire ne connaît pas le préimage et le rejette, renvoyant une erreur. Si cette erreur est reçue, le sondeur
+  sait que le chemin de paiement avait suffisamment de liquidité pour prendre en charge le paiement lorsqu'il a été envoyé, et qu'un
+  paiement réel envoyé le long du même chemin pour le même montant réussira probablement. Si une autre erreur est reçue, telle qu'une
+  erreur indiquant que l'un des sauts le long du chemin ne pouvait pas transmettre le paiement, un nouveau chemin peut être sondé avant
+  d'envoyer le paiement réel.
+
+   La sonde préalable au paiement ("prévol") peut être utile avec de petites sommes d'argent pour trouver des sauts qui rencontrent des
+   problèmes pouvant causer des retards. Si quelques centaines de sats (ou moins) restent bloqués pendant quelques heures, ce n'est pas
+   grave pour la plupart des dépensiers---mais si le montant total d'un paiement représentant une partie importante du capital d'un nœud
+   reste bloqué, cela peut être très ennuyeux. Il est également possible de sonder plusieurs chemins simultanément et d'utiliser les
+   résultats pour choisir le meilleur chemin quelques instants plus tard lors de l'envoi d'un paiement. {% assign timestamp="53:01" %}
+
+{% include references.md %}
+{% include linkers/issues.md v=2 issues="28492,119,738,28246,6311,6617,6686,2613,7994,2547,2534" %}
+[LND v0.17.0-beta.rc5]: https://github.com/lightningnetwork/lnd/releases/tag/v0.17.0-beta.rc5
+[news253 ark]: /fr/newsletters/2023/05/31/#proposition-d-un-protocole-de-gestion-des-joinpool
+[maxwell clock stop]: https://www.reddit.com/r/Bitcoin/comments/37fxqd/it_looks_like_blockstream_is_working_on_the/crmr5p2/
+[law cov post]: https://lists.linuxfoundation.org/pipermail/lightning-dev/2023-September/004092.html
+[law cov paper]: https://github.com/JohnLaw2/ln-scaling-covenants[towns cov]: https://lists.linuxfoundation.org/pipermail/lightning-dev/2023-September/004095.html
```

Comment:
```suggestion
[law cov paper]: https://github.com/JohnLaw2/ln-scaling-covenants
[towns cov]: https://lists.linuxfoundation.org/pipermail/lightning-dev/2023-September/004095.html
```

Reply:
Oh ! thanks for this correction @murchandamus